task_02.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
```

Comment:
Directions indicated that all work was supposed to take place in the same file (`task_01.py`)